src/components/Slider/Slider.tsx

```diff
@@ -35,17 +35,15 @@ export const Slider = () => {
     useEffect(() => {
         if (!sliderPictures?.length) return
 
+        switchingImg(indexPicture)
+
         const timeNext = setTimeout(() => setIndexPicture(preState =>
```

Comment:
setTimeout return as id, would be better change variable's name to `timeoutId`

Reply:
fixed